docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc

```diff
@@ -92,104 +36,194 @@ is an iteration of the last one, hence it improves the decision made by the
+model evaluation.
+
+If possible, prepare your input data such that it has less classes. A 
+{classanalysis} with many classes takes more time to run than a binary 
+{classification} job. The relationship between the number of classes and the 
+runtime is roughly linear.
 
+You might also need to {ref}/transforms.html[{transform}] your data to create a 
+{dataframe} which can be used as the source for {classification}.
+
+To learn more about how to prepeare your data, refer to 
+<<prepare-transform-data, the relevant section>> of the supervised learning 
+overview.
 
 [discrete]
-[[ml-inference-processor-class]]
-===== {infer-cap} processor
+[[dfa-classification-create-job]]
+== 4. Create a job
 
-include::ml-dfa-shared.asciidoc[tag=dfa-inference-processor]
+{dfanalytics-jobs-cap} contain the configuration information and metadata 
+necessary to perform an analytics task. You can create {dfanalytics-jobs} via 
+{kib} or using the {ref}/put-dfanalytics.html[create {dfanalytics-jobs} API]. 
+
+Select {classification} as the analytics type, then select the field that you 
+want to predict (the {depvar}). You can also include and exclude fields.
 
+To improve performance, consider using a small `training_percent` value to train 
+the model more quickly. It is a good strategy to make progress iteratively: run 
+the analysis with a small training percentage, then evaluate the performance. 
+Based on the results, you can decide if it is necessary to increase the 
+`training_percent` value.
 
 [discrete]
-[[ml-inference-aggregation-class]]
-===== {infer-cap} aggregation
+[[dfa-classification-start]]
+== 5. Start the job
 
-include::ml-dfa-shared.asciidoc[tag=dfa-inference-aggregation]
+You can start the job via {kib} or using the 
+{ref}/start-dfanalytics.html[start {dfanalytics-jobs}] API. A {classification} 
+job has the following phases: 
 
+include::ml-dfa-shared.asciidoc[tag=dfa-phases]
+
+After the last phase is finished, the job stops and the results are ready for 
+evaluation.
+
+include::ml-dfa-shared.asciidoc[tag=dfa-model-jvm]
 
 [discrete]
-[[dfa-classification-performance]]
-=== {classification-cap} performance
+[[ml-dfanalytics-classification-evaluation]]
+== 6. Evaluate and interpret the result
 
-As a rule of thumb, a {classanalysis} with many classes takes more time to run 
-than a binary {classification} process when there are only two classes. The 
-relationship between the number of classes and the runtime is roughly linear.
+include::ml-dfa-shared.asciidoc[tag=dfa-evaluation-intro]
 
-The runtime also scales approximately linearly with the number of involved 
-documents below 200,000 data points. Therefore, if you double the number of 
-documents, then the runtime of the analysis doubles respectively.
+You can measure how well the model has performed on your training data set by 
+using the `classification` evaluation type of the 
+{ref}/evaluate-dfanalytics.html[evaluate {dfanalytics} API]. The 
```

Comment:
Since some of the subsequent sections show screenshots from Kibana, I think it's worth mentioning somewhere in this intro that you don't have only the API available for this evaluation. For example:
```suggestion
{ref}/evaluate-dfanalytics.html[evaluate {dfanalytics} API] or by viewing the job results in {kib}. The 
```